builder.py

```diff
@@ -149,14 +149,10 @@ def BuildQueueCleanup(self, queue):
         queue.append('date')
         # Basic steps are:
         #   1. git stash (in each subproject)
-        #   2. git clean -fdx (in each subproject)
-        #   3. git fetch (in each subproject)
+        #   2. git fetch (in each subproject)
         queue.append('git stash')
         queue.append('git submodule foreach git stash')
         #
-        queue.append('git clean -fdx')
-        queue.append('git submodule foreach git clean -fdx')
-        #
         queue.append('git fetch')
         queue.append('git submodule foreach git fetch')
```

Comment:
This one is superfluous, a fetch in a superproject fetches the submodules too (which is nice!).

```
       --recurse-submodules[=yes|on-demand|no]
           This option controls if and under what conditions new commits of populated
           submodules should be fetched too. It can be used as a boolean option to
           completely disable recursion when set to no or to unconditionally recurse into
           all populated submodules when set to yes, which is the default when this option
           is used without any value. Use on-demand to only recurse into a populated
           submodule when the superproject retrieves a commit that updates the submodule’s
           reference to a commit that isn’t already in the local submodule clone.
```